db/migrate/20170408213508_create_featured_slides.rb

```diff
@@ -0,0 +1,9 @@
+class CreateFeaturedSlides < ActiveRecord::Migration[5.0]
+  def change
+    create_table :featured_slides do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->